docs/configs/docs3-1-0.js

```diff
@@ -25,19 +25,19 @@ export default {
                 children: [
                     {
                         title: 'Introduction',
-                        link: '/en-us/docs/dev/user_doc/about/introduction.html',
+                        link: '/en-us/docs/3.1.0/user_doc/about/introduction.html',
```

Comment:
you should keep the document in the old name, `docsdev.js`, it will auto change the file name during the sync process in `dolphinscheudler-website` repo, see more detail in https://github.com/apache/dolphinscheduler-website/blob/master/scripts/rsync_content.sh#L109